src/vivarium_public_health/risks/distributions.py

```diff
@@ -137,7 +137,7 @@ def ppf(self, x):
         if not np.allclose(1, np.sum(sorted_exposures, axis=1)):
             raise MissingDataError('All exposure data returned as 0.')
         exposure_sum = sorted_exposures.cumsum(axis='columns')
```

Comment:
Would it perhaps be even faster if you calculate `exposure_sum` once in `setup` instead of recomputing it in every call to `ppf`? IIUC, it does not change after setup.